test-requirements.txt

```diff
@@ -1,3 +1,2 @@
-requests==2.13.0
-future==0.14.3
+httpx==0.13.3
```

Comment:
As it's already in the `install_requires` what is the need to add it there too ?